postgres/problems/problem_09.md

```
@@ -1,21 +1,23 @@
 
 ## 説明
-ユーザ名が **`A`** さんが購入したことのある商品で、
-ユーザ名が **`B`** さんは 1 度も購入していない商品を抽出し、
-`id`, `name` を表示してください。
+`orders` テーブルをユーザごとに集計して注文数を求め、
+その結果を `users` テーブルと結合して、各ユーザの注文数をランキング形式で取得してください。
 
 ## 制約
-* ユーザ名は固定で `A`, `B` とすること
-* 結果は `id` 昇順で並べること
+* サブクエリで注文数を集計すること
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)